### PR TITLE
[GridRow] Add tests and update the CHANGELOG

### PR DESCRIPTION
**Pull request by nononoah**
### Summary

This PR adds a CHANGELOG note and tests for `GridRow`, but no more. Tests are loosely based off of those written for `EqualStack`.

I intend to merge this pr into #208 before merging both into main.

### CODE REVIEW

**Review by kyleve — APPROVED** [review]

File: BlueprintUI/Tests/GridRowTests.swift [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        //  Proportional children size to fit the available width. The tallest child is used as the measured height.
+        let gridRow = GridRow { row in
+            row.add(child: .proportional(1, TestElement(size: CGSize(width: 10, height: 5))))
+            row.add(child: .proportional(2, TestElement(size: CGSize(width: 25, height: 10))))
+        }
+        let constraint = SizeConstraint(width: .atMost(25), height: .atMost(100))
+        XCTAssertEqual(gridRow.content.measure(in: constraint), CGSize(width: 25.0, height: 10))
+    }
+
+    func test_layout() {
```
Comment (watt) [inline BlueprintUI/Tests/GridRowTests.swift:66]: Good coverage! Here's a couple more corner cases that might be useful:
- a row with only absolute children
- a child with proportion of zero
- a row that is too large for the available size (OK to have overflowing frames, but the proportional ones shouldn't be negative)

**Review by watt — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+```swift
+GridRow { row in
+  row.spacing = 8
+  row.verticalAlignment = .center
+  row.add(child: .absolute(authorLabel, 50))
+  row.add(child: .proportional(bodyLabel, 0.75))
+  row.add(child: .proportional(dateLabel, 0.25))
+}
+```
```
[the remark this answers is not in the document]
Comment (nononoah) [reply CHANGELOG.md:33]: Thanks for the tip. Updated.

File: BlueprintUI/Tests/GridRowTests.swift (hunk 1 above)
Comment (nononoah) [reply BlueprintUI/Tests/GridRowTests.swift:66]: Added a `test_layout_edgeCases()` method with these tests. Thanks for the suggestion.
